oemoflex/preprocessing.py

```diff
@@ -1005,6 +1007,10 @@ def update_hydro_reservoir(data_preprocessed_path, scalars):
         scalars,
         'EnergyConversion_VarOM_Electricity_Hydro_Reservoir') * 1e-3  # Eur/GWh -> Eur/MWh
 
+    # Apply auxilliary costs to the pump flow to avoid costless 'pump-and-spillover curtailment'
+    # Use string representation of the dict
+    element_df['input_parameters'] = dumps({"variable_costs": 0.00001})
```

Comment:
Could be defined as a default parameter.

Reply:
Unfortunately, this is more complex than we discussed. I will try it with hydro-reservoir only, to have a better view on what's going on before we discuss it in more detail.

Reply:
Found a solution with some limitation, see https://github.com/modex-flexmex/oemo-flex/pull/81#issuecomment-707567059